<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Laba3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="MyOpenGLWidget" name="widget" native="true"/>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>YRot</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox1">
        <property name="text">
         <string>vision</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ZRot</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="zRot">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Light</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Apro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="apro">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="xRot">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="yRot">
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>XRot</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSlider" name="size">
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">myopenglwidget.h</header>
   <container>1</container>
   <slots>
    <signal>xRotChanged(int)</signal>
    <signal>yRotChanged(int)</signal>
    <signal>zRotChanged(int)</signal>
    <signal>paramChanged(int)</signal>
    <signal>visionChanged(int)</signal>
    <signal>prgChanged(int)</signal>
    <signal>aproChanged(int)</signal>
    <slot>setXRot(int)</slot>
    <slot>setYRot(int)</slot>
    <slot>setZRot(int)</slot>
    <slot>setParam(int)</slot>
    <slot>setVision(int)</slot>
    <slot>setPrg(int)</slot>
    <slot>setApro(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>xRot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setXRot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yRot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setYRot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zRot</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setZRot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>641</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>size</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setParam(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>707</y>
    </hint>
    <hint type="destinationlabel">
     <x>914</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox1</sender>
   <signal>stateChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setVision(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>apro</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setApro(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>1085</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>widget</receiver>
   <slot>setPrg(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>729</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
